samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazorML.ConsoleApp/ModelBuilder.cs

```diff
@@ -0,0 +1,142 @@
+//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
+//*                                                                                       *
+//*****************************************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using SentimentRazorML.Model.DataModels;
+
+namespace SentimentRazorML.ConsoleApp
+{
+    public static class ModelBuilder
+    {
+        private static string TRAIN_DATA_FILEPATH = @"C:\Users\luquinta.REDMOND\Downloads\wikipedia-detox-250-line-data.tsv";
```

Comment:
Local path should be relative to the project.

Reply:
Hmm. good catch @justinormont . This was part of the auto-generated project. 

Question:

Should the auto-generated project hard-code the path where the data lives or should it copy the data to the auto-generated project, include it in the output directory and reference it from within the project? This way, if the user deletes the original data file, the project still works.

Reply:
Are you asking about how AutoML should handle the files? Or just for this example?

For AutoML in general, the files can be large and we wouldn't want to make another copy. 

For just this example, I'd recommend either a relative path to the dataset (to not duplicate the dataset in the repo), or have the example download a larger version from the CDN.

Reply:
Yep. I was asking about AutoML but avoiding duplicates makes sense. Thanks.

Reply:
Added data to console app and copied it to build directory. Usually, this would not be the suggested approach because both the console app and model class library are auto-generated but this was done for the purposes of this sample to be able to run after being cloned.